substrate/client/network/src/request_responses.rs

```diff
@@ -684,8 +619,8 @@ impl NetworkBehaviour for RequestResponsesBehaviour {
 							log::debug!(
 								target: "sub-libp2p",
 								"Failed to send response for {:?} on protocol {:?} due to a \
-								 timeout or due to the connection to the peer being closed. \
-								 Dropping response",
+								timeout or due to the connection to the peer being closed. \
+								Dropping response",
```

Comment:
Is this a `rustfmt` change?

Reply:
No, this is my change to avoid double whitespace in logs

Reply:
I think all whitespace in the beginning of multiline string literal continuations is ignored. My tests seem to [confirm this](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=6ddccd2eb1745da5783eafa5d16bf7d9).

Reply:
Hm, it is still odd to have it misaligned like that, it was probably in a different situation that I confused this with. Feel free to revert this change.